src/components/select/select.ts

```diff
@@ -365,6 +364,17 @@ export class Select extends BaseInput<string[]> implements AfterViewInit, OnDest
     });
   }
 
+  /**
+   * Close the select interface.
+   */
+  close() {
+    if (!(this._overlay && this.isFocus())) {
+      return;
+    }
+
+    this._overlay.dismiss();
+    this._overlay = undefined;
```

Comment:
do this only after dismiss happens. something like `.dismiss().then(() => {this._overlay = undefined})`